redaxo/src/addons/structure/pages/linkmap.php

```diff
@@ -9,6 +9,9 @@
 $clang = rex_request('clang', 'int');
 $clang = rex_clang::exists($clang) ? $clang : rex_clang::getStartId();
 
+$opener_input_field = preg_replace('/[^a-z0-9]/i', '', $opener_input_field);
```

Comment:
War mir nicht sicher was hier alles in addons genutt wird, daher hab ich mal zahlen und buchstaben erlaubt